Automattic/jetpack
# My Jetpack: Introduce card actions menu in Product Card

Closes #29676 

## Proposed changes:

* Add the `Menu` component and relative props.
* Improve the `ProductCard` story.

### Other information:


## Jetpack product discussion


N/A

## Does this pull request change what data or activity we track or use?


No.

## Testing instructions:






* Start `Storybook`.
* Navigate to `My Jetpack/Components/Product Card`, and select `With Menu` story.
* You should see the menu with the fixed option `Manage` and the appended one `Upload`.
* Navigate to the `Default` story. Nothing should change since the menu is hidden.
* Check the `My Jetpack` wp-admin page too. Nothing should change.

## Demo

![CleanShot 2023-03-30 at 17 15 42](https://user-images.githubusercontent.com/1663717/228954998-8e4df26f-098b-40f5-8265-fa54bf881048.png)
